### DISCUSSION TITLE
Sync third-party dependencies

### DISCUSSION THREAD

**Discussion by nagyatka** [moonrepo/moon#1543] [Q&A] [answered]
Hello moonrepo community!

I have recently started to try moonrepo, and I have a question for which I have not yet found a solution in the documentation.
In the past, I have encountered the problem that the versions of the third-party dependencies of monorepo projects can differ, which can lead to interesting bugs.
I thought that running the moon sync projects command would update the third-party dependency versions, but this did not happen. 
Is there a workaround for this problem?

**Comment by milesj** [comment, marked as answer]
moon doesn't get that granular at this time, but that kind of functionality is somewhat on our roadmap for the future. For now, you'd have to use a 3rd party tool.
